src/main/resources/templates/emailTemplate.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+</head>
+<body>
+Hello!
+<p>
+    You have one alert generated by detector "<span th:text="${detector}"></span>" for metric with key "<span th:text="${metricKey}"></span>"
+    and value "<span th:text="${metricValue}"></span>".
+</p>
+<p>More details on the alert message are:</p>
+<ul>
+    <li th:each="tag : ${metadata}" th:text="${tag}"></li>
+</ul>
+
+<p>Alert subscription rules can be access from <a href="">console</a>.</p>
```

Comment:
Spelling error: "access" should be "accessed" in this case.

Reply:
Even better, use an active voice:

"You can access alert subscription rules from the <a href="...">console</a>."

Reply:
+1 I just didn't want to get into a grammar and usage discussion when there's coding to be done 😄

Reply:
done